libs/designsystem/src/lib/components/modal/alert/alert.component.scss

```
@@ -1,14 +1,8 @@
 @import '../../../scss/utils';
 
-h3.title {
-  margin-top: size('m');
-  margin-bottom: size('m');
-  font-weight: font-weight('bold');
-}
-
 article {
   overflow: hidden;
-  padding: size('s');
+  padding: size('m') size('s') size('s');
```

Consider readability:
```suggestion
  padding: size('s');
  padding-top: size('m');
```